CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Profiles compatible with probgen programs now included in pipe output (#135).
 - Haplotype call plots now included in the pipe HTML report (#136).
 - Added offtarget module to count reads that map to off target loci in hg38 (#141).
+- Added typing rate and mapping rate information per marker  to the main pipe HTML report
+- Added marker detail HTML report
```

Comment:
By convention, we include the reference to the corresponding PR(s) for each change log entry. Of course, if you haven't created the PR yet, you probably won't know the number! But this is something you want to include in subsequent commits.

Reply:
Added that in and realized that in the off-target mapping merge request I put the issue # not the PR# so I also fixed that.